### DISCUSSION TITLE
Alma Gêmea

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#355] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/alma-gemea/

**Comment by the-coffee-monk** [comment]
O Ditto deve ser um dos Pokémon mais incompreendidos que eu conheço :pensive:

**Comment by one-eyed-willy** [comment]
Quem é esse Pikachu da 25 de Março?

**Comment by maryon-telvanni** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/tirinhas/alma-gemea/#comment_1173957632>):

> Quem é esse Pikachu da 25 de Março?

![null](<https://i.gifer.com/origin/c7/c79ef614a29ad87b107ba74c444369aa.gif>)

**Comment by the-coffee-monk** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/tirinhas/alma-gemea/#comment_1173957632>):

> Quem é esse Pikachu da 25 de Março?

É o Mimikyu

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/tirinhas/alma-gemea/#comment_1173957632>):

> Quem é esse Pikachu da 25 de Março?

![null](<https://i.imgur.com/DKK9oof.jpg>)

**Comment by tali-zorah** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/tirinhas/alma-gemea/#comment_1173957632>):

> Quem é esse Pikachu da 25 de Março?

*Keelah!* Tadinho, Willy! *O background* dele é tão triste :pensive:
